src/main/java/com/github/mc1arke/sonarqube/plugin/server/pullrequest/ws/action/ValidateAction.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2020 Michael Clarke
```

Comment:
The year should be 2021, and I'd suggest you put your name unless you're granting me ownership of your code.